watchdog/.env

```diff
@@ -0,0 +1,8 @@
+
+# DFX CANISTER ENVIRONMENT VARIABLES
+DFX_VERSION='0.13.1'
+DFX_NETWORK='local'
+CANISTER_ID_watchdog='rrkah-fqaaa-aaaaa-aaaaq-cai'
+CANISTER_ID='rrkah-fqaaa-aaaaa-aaaaq-cai'
+CANISTER_CANDID_PATH='/Users/maksym/dfinity/bitcoin-canister-w/watchdog/canister/candid.did'
+# END DFX CANISTER ENVIRONMENT VARIABLES
```

Comment:
I assume you don't need to merge these, given that the dfx version is already defined in `dfx.json` and that some variables are specific to your local setup?

Reply:
Done.